<launch>
    <arg name="image_topic" default="camera/image_rect"/>
    <arg name="detection_topic" default="/detection_image"/>
    <arg name="model"/>

    <node pkg="rr_ml" type="run_seg" name="run_seg" output="screen" respawn="false" required="true">
        <param name="model_path" type="string" value="$(find rr_ml)/models/segmentation/$(arg model).h5" />
        <param name="image_topic" type="string" value="$(arg image_topic)"/>
        <param name="detection_topic" type="string" value="$(arg detection_topic)"/>
        <param name="polling_rate" type="double" value="500"/>
    </node>
</launch>
